src/resolvers/Mutation/signUp.ts

```diff
@@ -112,6 +114,10 @@ export const signUp: MutationResolvers["signUp"] = async (_parent, args) => {
     password: hashedPassword,
   });
 
+  if (createdUser.email === LAST_RESORT_SUPERADMIN_EMAIL) {
```

Comment:
Create a variable createdUser, store in it result of creating a user. Create the user conditionally(checking whether the email is LAST_RESORT_SUPERADMIN_EMAIL), and assign the value to createdUser.